build-1es-pipeline.yaml

```diff
@@ -1024,6 +1006,27 @@ extends:
                       echo "##vso[task.setvariable variable=azcopy_github_tag]$azcopy_github_tag"
                     displayName: 'AzCopy Tag Name'
 
+                  - task : DownloadPipelineArtifacts@2

```

Comment:
I think this should be not plural, when I try to run this it says step does not exist